### PR TITLE
fix: `file_format`

### PR DESCRIPTION
**Pull request by joscha**
Follow-up to https://github.com/Snowflake-Labs/terraform-snowflake-snowpipe-aws/pull/24

This also adds the suggested comment in https://github.com/Snowflake-Labs/terraform-snowflake-snowpipe-aws/pull/24#discussion_r1777901304

cc @sfc-gh-pkommini

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,6 @@ Terraform module for creating Snowpipe to ingest data from AWS S3 bucket.
...
 | --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | create_stage          | Default is "true". Set to "false" to use an existing stage and not creaete a new one.                                                                                                                                                          |
-| file_format           | Default is "JSON". Source file format.                                                                                                                                                                                                         |
+| file_format           | Default is "TYPE = JSON". Source file format.                                                                                                                                                                                                         |
```
Comment (joscha) [inline README.md:24]:
```suggestion
| file_format           | Default is "TYPE = JSON NULL_IF = []". Source file format and options.                                                                                                                                                                                                         |
```

**Comment by joscha** [comment]
@sfc-gh-pkommini a merge of this and a new version tag would be amazing 🙏 , please?

**Review by sfc-gh-pkommini — APPROVED** [review]
lgtm

**Comment by sfc-gh-pkommini** [comment]
> @sfc-gh-pkommini a merge of this and a new version tag would be amazing 🙏 , please?

Thanks a lot for your contributions. Apologize for the delay.

**Comment by joscha** [comment]
no worries, thank you! Did you mean to release a `2.1.0`? I expected a `0.25.0`? :)

**Comment by sfc-gh-pkommini** [comment]
@joscha oh wow. You're right! Sorry about that. I was going to bump up the minor version so something like `0.3.0`. Will fix now.

**Pull request merged**